Samples/iOS-Swift/iOS-SwiftUITests/UIEventTests.swift

```diff
@@ -0,0 +1,41 @@
+import XCTest
+
+class UIEventTests: XCTestCase {
```

Comment:
`m`: The name should reflect what it is testing. It's hard otherwise to find out about the correlation between this UITest and `SentryBreadcrumbTracker`.
```suggestion
class UIEventBreadcrumbTests: XCTestCase {
```